### DISCUSSION TITLE
add phpmyadmin to laragon

### DISCUSSION THREAD

**Discussion by ghost** [leokhoa/laragon#393] [General]
go to https://www.phpmyadmin.net/downloads/ and download the phpmyadmin file:
![image](https://user-images.githubusercontent.com/67982454/216743493-cecfb236-f6ee-423b-a129-f2a6a09d5bb2.png)

extract the file in the following path C:\laragon\etc\apps in a folder called phpMyAdmin
![image](https://user-images.githubusercontent.com/67982454/216743911-74777703-166c-4670-975e-a38d6a906d2b.png)

start the services again and that's it!!

![image](https://user-images.githubusercontent.com/67982454/216743789-f7b76739-0922-4acf-b643-1d3377463716.png)
![image](https://user-images.githubusercontent.com/67982454/216743826-138cd012-746c-4022-9c79-5a3f65f4442c.png)

**Comment by rescenic** [comment]
Thank you very much for the explanation.

**Comment by dontletthesamedogbiteyoutwice** [comment]
Never mind ...

I 'root'ed it out

Argh...
//edit
Works now. Just have to find out how to not require a password.

@LeninCarabali 

I extracted all the files to D:\laragon\etc\apps\phpMyAdmin but when I try and run it all the browser shows is http://localhost/phpmyadmin/ **with a blank screen :-(**

**Comment by MGeurts** [comment]
https://www.kreaweb.be/laragon-add-phpmyadmin/

**Comment by Lycantant** [comment]
Open phpMyAdmin without password.

/* Authentication type */
`// $cfg['Servers'][$i]['auth_type'] = 'cookie';`
`$cfg['Servers'][$i]['auth_type'] = 'config';`
`$cfg['Servers'][$i]['username'] = 'root';`
`$cfg['Servers'][$i]['password'] = '';`

**Comment by coycoylaniba** [comment]
i downloaded phpmyadmin from https://www.phpmyadmin.net/downloads/ and then extract and rename it to `phpMyAdmin` and copied it to `etc/apps` then restart apache and mysql but then i'm gettings this:

![image](https://github.com/leokhoa/laragon/assets/47078705/e0e26350-6f2a-4789-a4e6-48fe7964116f)

**Comment by MGeurts** [comment]
Just follow this guide : [https://www.kreaweb.be/laragon-add-phpmyadmin/](https://www.kreaweb.be/laragon-add-phpmyadmin/) and you'll be good.

**Comment by iftakharul-xs** [comment]
Just goto Menu -> Tools -> Quick Add -> click phpmyadmin 
automatically added phpmyadmin no extra hack needed. 

![image](https://github.com/leokhoa/laragon/assets/115061454/4d3d8dd9-eae0-4c37-b9e1-83798c69351d)

**Reply by MGeurts** [reply]
Yes. But then you have an older version. Follow my guide and you will get the latest version.
It's not a hack. It's just correcting the link to the latest version.

[https://www.kreaweb.be/laragon-add-phpmyadmin/](https://www.kreaweb.be/laragon-add-phpmyadmin/)
